turbinia/lib/recipe_helpers.py

```diff
@@ -166,4 +171,58 @@ def validate_recipe(recipe_dict):
         return (False, message)
       tasks_with_recipe.append(recipe_item)
 
-  return (True, '')
\ No newline at end of file
+  return (True, '')
+
+
+def validate_recipe_conditions(args):
```

Comment:
Looking at this again, should we move these checks into `create_recipe()`?  The problem with keeping them here is that this method is pretty specific to the argument objects that are set up by `turbiniactl` and so when other clients are setting up recipes these checks won't be able to be run easily without having this same arg input.  I think most of these arguments are already passed to `create_recipe()` though.

Reply:
yeah, good point, that's probably better.